proposals/description/fip_82b.ts

```diff
@@ -0,0 +1,480 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_82b: ProposalDescription = {
+  title: 'FIP-82b: Authorise the TribalCouncil with necessary roles',
+  commands: [
+    /////// Transfer all non-major role admins to the ROLE_ADMIN, to allow TribalCouncil to manage ////////
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xf0b50f04623eeaacfa1f202e062a3001c925a35c6b75d6903e67b43f44bbf152', // PARAMETER_ADMIN
+        '0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096' // ROLE_ADMIN
+      ],
+      description: 'Transfer PARAMETER_ADMIN role admin to ROLE_ADMIN'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xc7be67d109cc181c862ac2eaed91ab54817910d73264a8ad5d87592516929a15', // MINOR_ROLE_ADMIN
+        '0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096' // ROLE_ADMIN
+      ],
+      description: 'Transfer MINOR_ROLE_ADMIN role admin to ROLE_ADMIN'
```

Comment:
I believe MINOR_ROLE_ADMIN was an old role idea that was meant to be equal to ROLE_ADMIN. If you don't see a need for this role, we can remove it

Reply:
Yep, hasn't been created yet either. Removing